sympy/assumptions/tests/test_query.py

```diff
@@ -2046,17 +2046,6 @@ def test_compute_known_facts():
     s = compute_known_facts(known_facts, known_facts_keys)
 
 
-@slow
-def test_known_facts_consistent():
-    """"Test that ask_generated.py is up-to-date"""
-    from sympy.assumptions.ask import get_known_facts, get_known_facts_keys
-    from os.path import abspath, dirname, join
-    filename = join(dirname(dirname(abspath(__file__))), 'ask_generated.py')
-    with open(filename, 'r') as f:
-        assert f.read() == \
-            compute_known_facts(get_known_facts(), get_known_facts_keys())
```

Comment:
This line was giving assertion error

```python
assert f.read() == compute_known_facts(get_known_facts(), get_known_facts_keys())
```

It has no Boolean expressions. I wanted to create an Issue for this.